modules/apps/object/object-api/src/main/java/com/liferay/object/model/ObjectDefinition.java

```diff
@@ -61,6 +61,8 @@ public Class<ObjectDefinition> getTypeClass() {
 
 	public String getExtensionDBTableName();
 
+	public String getL10nDBTableName();
```

Comment:
yes, 

I put l10n because it's the abreviation of localization and also to follow the pattern followed in other places abreviating internacionalization to i18n, 

but, I should be consistent and named DynamicObjectDefinitionLocalizationTable to DynamicObjectDefinitionL10nTable

Reply:
done [here](https://github.com/liferay-objects/liferay-portal/pull/1913/commits/7f61d38e231e83a79a47b6bd239dcb74ab46ee7b)